tests/foreman/cli/test_setting.py

```diff
@@ -36,29 +40,32 @@ def test_negative_update_hostname_with_empty_fact(self):
         :caseautomation: notautomated
         """
 
-    @stubbed()
     @tier1
     def test_positive_update_hostname_prefix_without_value(self):
         """Update the Hostname_prefix settings without any string(empty values)
 
         :id: a84c28ea-6821-4c31-b4ab-8662c22c9135
 
         :expectedresults: Hostname_prefix should be set without any text
-
-        :caseautomation: notautomated
         """
+        setting = Settings.set({'name': "discovery_prefix", 'value': ""})
+        self.assertIn("", setting[0])
```

Comment:
This specific assertion doesn't really verify anything, "emptiness" is present in the string for sure :)
Anyway, pls avoid verifying messages. Prefer verifying actual value instead. Should be smth like this:
```python
discovery_prefix = Settings.list({'search': 'name=discovery_prefix'})[0]
self.assertEqual('', discovery_prefix['value'])

Reply:
Comment applicable to all the tests